starter/src/test/java/com/starter/fullstack/rest/ProductControllerTest.java

```diff
@@ -25,74 +26,77 @@
 @RunWith(SpringRunner.class)
 public class ProductControllerTest {
 
-  @Autowired
-  private MockMvc mockMvc;
+    @Autowired
```

Comment:
It doesn't look like this file needed to be changed. It also looks like it was just a bunch of indents that changed. Be careful with this type of stuff because usually there will be file rules for your project such as indent size. I believe this project is setup with a indent size of 2 spaces. It would probably helpful to set your Intellij indent size to 2.

Reply:
Changed all indentation to 2
![Screenshot 2024-07-09 at 19 00 31](https://github.com/K-Shields1994/fullstack-starter/assets/120108762/4d874348-d1c7-4b56-93b0-70e4f7457f12)